packages/snap/src/rpc/generateTransactionPayload.ts

```diff
@@ -6,7 +6,8 @@ import { getAddress } from './getAddress';
 export async function generateTransactionPayload(
```

Comment:
remove this method we don't need `generateTransactionPayload` in the snap, it's not standard, You won't find it in the polkadotjs or other wallet. 
the client (example/demo) code should submit tx via `SignerPayloadJSON`,

Reply:
That's a fork, it was already there and this snap (from polkadot) is on Snap store haha. But I agree, no need to be there.